### ISSUE TITLE
GTA: San Andreas - The Definitive Edition crashing with fatal error message if using the HDR flags.

### ISSUE THREAD

**Issue by Hadrianneue** [ValveSoftware/gamescope#1625]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using any gamescope patches or a forked version of gamescope?

- [X] The issue occurs on upstream gamescope without any modifications

### Current Behavior

Game simply crashes when i try to use wayland if i pass the flag --hdr-enabled ~~with or without additional options like HDR or force-grab-cursor,~~ steam discussions shows that is happening as of today on the steam deck as well given the latest game update

### Steps To Reproduce

just try it, it will crash with a fatal error message

### Hardware information

```markdown
- Distro: endeavouros
- CPU: 5700X3D
- GPU: RX 6700 XT
- Driver Version: Mesa-git
```

### Software information

```markdown
- Desktop environment: KDE 6.2
- Session type: wayland
- Gamescope version: 3.15.9-21-g7dd1bcd9 (gcc 14.2.1)
- Gamescope launch command(s): `ENABLE_HDR_WSI=1 
gamescope -f -w 2560 -h 1440 --force-grab-cursor --hdr-enabled --hdr-debug-force-output -r 360 -- env ENABLE_GAMESCOPE_WSI=1 DISABLE_HDR_WSI=1 %command%`
```

### Which gamescope backends have the issue you are reporting?

- [X] Wayland (default for nested gamescope)
- [ ] DRM (default for embedded gamescope, i.e. gamescope-session)
- [ ] SDL
- [ ] OpenVR

**Comment by matte-schwartz** [comment]
what about dropping all the HDR args and using `ENABLE_GAMESCOPE_WSI=0 gamescope -f -w 2560 -h 1440 --force-grab-cursor -r 360 -- %command%` on wayland?

**Comment by Hadrianneue** [comment]
> what about dropping all the HDR args and using `ENABLE_GAMESCOPE_WSI=0 gamescope -f -w 2560 -h 1440 --force-grab-cursor -r 360 -- %command%` on wayland?

you are right actually, without HDR it works correctly, let me change the description then

**Comment by matte-schwartz** [comment]
If that command I gave you worked, the problem is likely with the Gamescope WSI layer. It gets used by default on Wayland desktop and with `--hdr-enabled` which uses the WSI layer, but can be disabled with that first environment variable I gave you. 

WSI is enabled by default for the Steam Deck now which would also explain other people experiencing the issue on Deck.

**Comment by Hadrianneue** [comment]
> If that command I gave you worked, the problem is likely with the Gamescope WSI layer. It gets used by default on Wayland desktop and with `--hdr-enabled` which uses the WSI layer, but can be disabled with that first environment variable I gave you.
> 
> WSI is enabled by default for the Steam Deck now which would also explain other people experiencing the issue on Deck.

never mind, so setting alone (without gamescope) the env var DXVK_HDR=1 triggers this crash, its probably DXVK related, ENABLE_GAMESCOPE_WSI=0 didn't make any difference

**Comment by Hadrianneue** [comment]
> If that command I gave you worked, the problem is likely with the Gamescope WSI layer. It gets used by default on Wayland desktop and with `--hdr-enabled` which uses the WSI layer, but can be disabled with that first environment variable I gave you.
> 
> WSI is enabled by default for the Steam Deck now which would also explain other people experiencing the issue on Deck.

almost certain its DXVK related, i think i had another UE4 game with a similar issues, running with %command% -dx12 launches normally, even with the HDR options, but HDR doeS not work, thanks, i'm going to submit this issue on dxvk issue tracker

**Comment by matte-schwartz** [comment]
Looks like this has been fixed in dxvk https://github.com/doitsujin/dxvk/pull/4450

**Comment by Hadrianneue** [comment]
> Looks like this has been fixed in dxvk [doitsujin/dxvk#4450](https://github.com/doitsujin/dxvk/pull/4450)

let me close it then, thank you
